doc/source/docs/user_guide/projections.rst

```diff
@@ -80,24 +80,25 @@ Re-projecting is the process of changing the representation of locations from on
 
 .. ipython:: python
 
+    import geodatasets
+
     # load example data
-    world = geopandas.read_file(geopandas.datasets.get_path('naturalearth_lowres'))
+    usa = geopandas.read_file(geodatasets.get_path('geoda nat regimes'))
 
     # Check original projection
     # (it's Plate Carrée! x-y are long and lat)
-    world.crs
+    usa.crs
 
     # Visualize
-    ax = world.plot()
-    @savefig world_starting.png
+    ax = usa.plot()
+    @savefig usa_starting.png
     ax.set_title("WGS84 (lat/lon)");
 
-    # Reproject to Mercator (after dropping Antartica)
-    world = world[(world.name != "Antarctica") & (world.name != "Fr. S. Antarctic Lands")]
-    world = world.to_crs("EPSG:3395") # world.to_crs(epsg=3395) would also work
-    ax = world.plot()
-    @savefig world_reproj.png
-    ax.set_title("Mercator");
+    # Reproject to Albers contiguous USA
+    usa = usa.to_crs("ESRI:102003")
```

Comment:
Not 100% sure but should these be using standard EPSG codes rather than ESRI codes? (I remember some issue a while back around these, I just don't have a sense of if if only certain esri codes are problematic or if it is more widespread.

Reply:
I don't see a reason why not to use this one. It is probably also good to illustrate that not only EPSG is supported.